cmd/agent/daemon/state/netflow_pipeline.go

```diff
@@ -29,7 +29,7 @@ func (c *Controller) getClusterInfo(ctx context.Context) (*clusterInfo, error) {
 		resp, err := c.kubeClient.GetClusterInfo(ctx, &kubepb.GetClusterInfoRequest{})
 		if err != nil {
 			c.log.Warnf("getting cluster info: %v", err)
-			sleep(ctx, 2*time.Second)
+			sleep(ctx, 3*time.Second)
```

Comment:
Why was the sleep increased?

Reply:
During redeployment/install and on tilt it can take some time for controller to running. Increased a bit to make less warnings. Can revert.

Reply:
No need, I was just curious.